sentinel-core/transport/command/handler/vo/statistic_node_vo.h

```diff
@@ -0,0 +1,69 @@
+#include <memory>
+#include <string>
+
+#include "sentinel-core/statistic/node/cluster_node.h"
+
+namespace Sentinel {
+namespace Transport {
+
+class StatisticNodeVO {
```

Comment:
VO? What means?

Reply:
VO = view object for web controller (according to Alibaba Java Code Guidelines)